src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting_NetFx/RazorFormattingTest.cs

```diff
@@ -106,7 +106,8 @@ private void IncrementCount()
                             }
                         }
                     </boo>
-                    """);
+                    """,
+            inGlobalNamespace: inGlobalNamespace);
```

Comment:
@davidwengier 

I've tried testing formatting on component in global namespace (debugged through it to verify the generated C# code really doesn't contain the `namespace` block). It works. But I'm not sure if this is the correct place or correct way to test this.

I've also tried this in VS - and the design-time doc seems to be using `namespace __GeneratedComponent` which is wrong but at least formatting isn't broken :D I will try to see if that can be fixed as well (without breaking formatting)

Reply:
If the test is generating the code you expect, then great. Perhaps my fears are unfounded, or its only certain edge cases that would be affected. Either way, adding the infrastructure for us to test it, like you have, will be very valuable so thank you very much for that.

Reply:
I ended up doing what you suggested to fix formatting. Thanks!